apps/web/src/app/infonaytto/layout.tsx

```diff
@@ -0,0 +1,64 @@
+"use client";
+import React, { useRef } from "react";
+import { Inter, Roboto_Mono } from "next/font/google";
+import { cn } from "../../lib/utils.ts";
+import "../globals.css";
+import { InfoScreenHeader } from "../../components/infoscreen/infoscreen-header";
+
+const inter = Inter({
+  subsets: ["latin"],
+  variable: "--font-inter",
+});
+const robotoMono = Roboto_Mono({
+  subsets: ["latin"],
+  variable: "--font-roboto-mono",
+});
+
+export default function ScreenLayout({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  const containerRef = useRef(null);
+
+  const enterFullScreen = () => {
+    if (containerRef.current) {
+      if (containerRef.current.requestFullscreen) {
+        containerRef.current.requestFullscreen();
+      } else if (containerRef.current.webkitRequestFullscreen) {
+        containerRef.current.webkitRequestFullscreen(); // For Safari
+      } else if (containerRef.current.msRequestFullscreen) {
+        containerRef.current.msRequestFullscreen(); // For older browsers
+      }
```

Comment:
Most browsers since 2020 have support for the standard version, and Safari from March 2023 *plus* this feature is for "internal" use so maybe we could drop this compatibility check? [Source](https://developer.mozilla.org/en-US/docs/Web/API/Element/requestFullscreen#browser_compatibility)

Then we could change the `containerRef` to be typed `useRef<HTMLBodyElement>(null)` which probably resolves most type errors.